test/oli_web/live/progress/student_resource_view_live_test.exs

```diff
@@ -0,0 +1,108 @@
+defmodule OliWeb.Progress.StudentResourceViewLiveTest do
+  use ExUnit.Case
+  use OliWeb.ConnCase
+
+  import Phoenix.LiveViewTest
+  import Oli.Factory
+
+  defp live_view_student_resource_view_route(section_slug, user_id, resource_id) do
+    Routes.live_path(
+      OliWeb.Endpoint,
+      OliWeb.Progress.StudentResourceView,
+      section_slug,
+      user_id,
+      resource_id
+    )
+  end
+
+  defp setup_section_resource_access(instructor) do
+    {:ok, section: section, unit_one_revision: _unit_one_revision, page_revision: page_revision} =
+      section_with_assessment(%{})
+
+    student = insert(:user)
+
+    enroll_user_to_section(student, section, :context_learner)
+    enroll_user_to_section(instructor, section, :context_instructor)
+
+    resource_access =
+      insert(:resource_access,
+        user: student,
+        resource: page_revision.resource,
+        section: section,
+        score: 1,
+        out_of: 2
+      )
+
+    [section_slug: section.slug, student_id: student.id, resource_id: resource_access.resource_id]
+  end
+
+  describe "student resource view" do
+    setup [:instructor_conn]
+
+    test "score must be less or equal to out_of value", %{
+      conn: conn,
+      instructor: instructor
+    } do
+      [section_slug: section_slug, student_id: student_id, resource_id: resource_id] =
+        setup_section_resource_access(instructor)
```

Comment:
NIT: We can add this function to the describe `setup` since is something all tests will share, wdyt?